include/lsst/afw/detection/GaussianPsf.h

```diff
@@ -62,6 +62,9 @@ class GaussianPsf : public afw::table::io::PersistableFacade<GaussianPsf>, publi
     /// Polymorphic deep copy; should usually be unnecessary because Psfs are immutable.
     virtual std::shared_ptr<afw::detection::Psf> clone() const;
 
+    /// Return a clone with specified kernel dimensions
+    virtual std::shared_ptr<afw::detection::Psf> resized(int width, int height) const;
```

Comment:
See previous comments (on related PRs) about using `override` instead of `virtual`.